js/controllers/grid/users/stageParticipant/form/StageParticipantNotifyHandler.js

```diff
@@ -110,6 +119,38 @@
 
 
 	/**
+	 * Update the enabled/disabled and checked state of the recommendOnly checkbox.
+	 */
+	$.pkp.controllers.grid.users.stageParticipant.form.
+			StageParticipantNotifyHandler.prototype.updateRecommendOnly = function() {
+
+		var $form = this.getHtmlElement(),
+				$filterUserGroupId = $form.find('input[name=\'userGroupId\']'),
+				$checkbox = $('[id^=\'recommendOnly\']'),
+				$checkboxDiv = $('div.recommendOnlyClass'),
```

Comment:
You can use the `$('selector', $form)` approach here. Or `$form.find('selector')`. Either one works, but it's good to restrict the DOM search if you can.

Reply:
Also, let's call `recommendOnlyClass` something like `recommendOnlyWrapper`. And remove the `div.` from the selector so that we can change the element type down the line without breaking anything.